packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -342,7 +342,7 @@ class FlutterDriver {
     Future<Health> checkHealth() async {
       try {
         // At this point the service extension must be installed. Verify it.
-        return await driver.checkHealth();
+        return driver.checkHealth();
```

Comment:
humm, the `await` removal is not correct here in a try-catch section. I need to update the lint.